ux/designs/describedatastep/describedatastep.md

```diff
@@ -0,0 +1,64 @@
+# Describe Data Step
+
+Users can add a "Describe Data" step to an integration flow that declares the current data type.
+
+## Add a Step
+
+![Image of add a step](img/describeData_addaStep.png)
+
+1. User selects “Describe Data” as the step to declare data type.
+
+## Configure Describe Data Step
+
+![Image of configure describe data step](img/describeData_configStep_upload.png)
+
+1. User first needs to select schema type. This is a single select with 4 options.
+  - JSON schema
+  - JSON instance document
+  - XML schema
+  - XML instance document
+
+2. Then user can import schema by either “Upload a file” or “Use a URL”.
+
+    Upload file validation:
+    - JSON Schema - .json
+    - JSON instance - .json
+    - XML Schema - .xsd
+    - XML instance - .xml
+
+3. Specify Root Element field is disabled until there’s a valid schema file.
+
+![Image of file uploading](img/describeData_configStep_uploading.png)
+
+1. Show a spinner when uploading the file.
+
+2. When a file is in the upload process, the drop file zone and choose file button is disabled.
+
+![Image of upload error](img/describeData_configStep_uploadError.png)
+
+1. If file uploaded isn’t a supported file type, display error message in form of inline notification.  
+
+  E.g. If user selects JSON schema in the first step, an uploaded file should be in .json format.
+
+  Upload file validation:
```

Comment:
add an empty line which helps markdown to format this as a list.

Reply:
Are we expecting Citizen Integrators to be able to setup non-http routes, since that's the only way they will be able to access anything non-https outside OpenShift?